### ISSUE TITLE
Documentation is missing

### ISSUE THREAD

**Issue by djdomi** [DanielAdolfsson/ndppd#60]
Hi

I would like to tell you that this is a great project, the only problem is, you can't find useful tutorials to understand the usage of your software.

Can you please create some docs and examples?

**Comment by DanielAdolfsson** [comment]
You mean like a TL;DR? Otherwise, you should have plenty of information here.

https://github.com/DanielAdolfsson/ndppd/blob/1.0-devel/ndppd.conf-dist
https://github.com/DanielAdolfsson/ndppd/blob/1.0-devel/ndppd.conf.5.adoc
https://github.com/DanielAdolfsson/ndppd/blob/1.0-devel/ndppd.8.adoc

**Comment by djdomi** [comment]
yes, but why cant you create a full example conf? even with any available parameters and or add a hint into the conf header use man ndppd.conf?

**Comment by djdomi** [comment]
more over, it would be greatly for any users to see a full setup as howto on your project page, even if i nees to sponsor this with a euro or two ;)
